groups/groups.yaml

```diff
@@ -144,11 +144,7 @@ groups:
       - k8s-infra-rbac-k8s-io-prod@kubernetes.io
       - k8s-infra-rbac-perfdash@kubernetes.io
       - k8s-infra-rbac-publishing-bot@kubernetes.io
-      - k8s-infra-rbac-slack-event-log@kubernetes.io
-      - k8s-infra-rbac-slack-moderator@kubernetes.io
-      - k8s-infra-rbac-slack-welcomer@kubernetes.io
-      - k8s-infra-rbac-slackin@kubernetes.io
-
+      - k8s-infra-rbac-slack-infra@kubernetes.io
   - email-id: k8s-infra-alerts@kubernetes.io
```

Comment:
nit: a space between those two lines ?

Reply:
Great catch, I don't know how I missed it

Reply:
Done!